heartbeat/docs/monitors/monitor-browser.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[monitor-browser-options]]
+=== Browser options
+
+Also see <<monitor-options>>.
+
+The options described here configure {beatname_uc} to run the synthetic
+monitoring test suites via Synthetic Agent on the chromium browser.
+
+Example configuration:
+
+[source,yaml]
+----
+- type: browser
+  id: synthetic-inline-suites
+  name: Elastic website
+  schedule: '@every 1m'
+  source:
+    inline:
+      script: |-
+        step("load homepage", async () => {
+          await page.goto('https://www.elastic.co');
+        });
+        step("hover over products menu", async () => {
+          await page.hover('css=[data-nav-item=products]');
+        });
+----
+
+[float]
+[[monitor-browser-source]]
+==== `Source`
+
+Details about the how to run the synthetic test suites.
```

Comment:
the wording here is a bit odd. The source field is less about how to run the synthetics test suites then where. Maybe this should say: "Where to find the test to run". @dedemorton thoughts?